source/dub/project.d

```diff
@@ -1911,7 +1911,7 @@ final class SelectedVersions {
 
 /// The template code from which the test runner is generated
 private immutable TestRunnerTemplate = q{
-module dub_test_root;
+deprecated module dub_test_root;
```

Comment:
Maybe add an explanation here so a person looking at it is not puzzled as to why we do this.

Reply:
Done